repository-data/webfiles/src/main/resources/site/freemarker/common/macro/award.ftl

```diff
@@ -1,11 +1,47 @@
 <#ftl output_format="HTML">
 
-<#-- @ftlvariable name="section" type="uk.nhs.digital.website.beans.Award" -->
-
 <#include "../../include/imports.ftl">
 
-  <#macro award section>
-    <div class="award--div">
-      <h3>${section.awardname}</h3>
+  <#macro award awards idsuffix firstname>
+    <#if awards?has_content>
+    <div id="award-${slugify(idsuffix)}" class="award--div article-section">
+      <h2>Awards</h2> 
+
+        <p>
+          <#if awards?has_content >
+            ${firstname} has won awards for:
+            <ul>
+              <#list awards as personaward>
+                <div>
+                  <#assign awardnameDisplay = personaward.awardname />
+                  <#if personaward.awardlink?has_content>
+                    <li><a href="${personaward.awardlink.link}" onClick="logGoogleAnalyticsEvent('document click','Event','${personaward.awardlink.link}');" onKeyUp="return vjsu.onKeyUp(event)" title="${awardnameDisplay}"><span itemprop="award">${awardnameDisplay}</span></a></li>
```

Comment:
With the GA tags, the 2nd param is the Category, and the doc type is usually used here i.e. Publication, Event, Data set.  For this, as its a compound related to the Person doc type, could replace "Event" with "Person". Same applies to other GA tags in this PR which are related to Person.

Reply:
good point - I will replace

Reply:
solved by: https://github.com/NHS-digital-website/hippo/commit/4b1e9ee5a5497df9cca0f908a247b4bac8ad4d1a